### PR TITLE
Bug/programs landing page layout

### PR DESCRIPTION
**Pull request by jimjamesjimathy**
## Description

We started by using the flexbox method to give an even amount of space to the left and right sides of all Programs components as well as resizing, aligning and adding correct fonts to all rendered text on the page. In some cases, the height of the element also needed adjustment to display the content correctly, so we also adjusted each element accordingly.

Fixes # ( Over-lapping text and un-centered images on page )

## Loom Video

https://www.loom.com/share/daeff8089c34490ca4c9effcd0c8e8eb

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] I have made corresponding changes to the documentation if necessary (optional)
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

File: src/components/pages/LandingPrograms/CoderSitters.js [hunk 1]
```diff
@@ -8,50 +8,52 @@ export default function CoderSitters() {
...
-            <Text className="pl__na__age">Ages 6 and Up.</Text>
-          </div>
+            {/* BABYSITTER + INSTRUCTOR */}
+            <div className="pl__na__top__mid__container">
+              <Title className="pl__na__sitter__title__container" level={2}>
+                THINK BABY SITTER + CODING INSTRUCTOR.
+              </Title>
+
+              {/* INFO PARAGRAPH */}
+              <Text className="pl__na__sitter__text__container">
```
Comment (dstrazzeri) [inline src/components/pages/LandingPrograms/CoderSitters.js:32]: In PR reviews Edd mentioned there are ways to get around using explicit <br/> tags by using relative units for sizing within text containers.

**Review by dstrazzeri — APPROVED** [review]
Overall, super solid PR. I really appreciate your use of descriptive commit messages. And great use of flexbox, the layout is a huge improvement.

File: src/styles/LandingPageStyles/index.less [hunk 2]
```diff
@@ -15,6 +15,7 @@
 
 .landing-main-title {
   font-family: 'Montserrat', cursive !important;
```
Comment (dstrazzeri) [inline src/styles/LandingPageStyles/index.less:17]: Paul & Edd also mentioned trying to stay away from using !important tags, so this is something as a team we should look into and discuss as I know it also used on previously written code.

File: src/components/pages/LandingPrograms/CoderYoga.js [hunk 3]
```diff
@@ -26,13 +26,15 @@ export default function CoderYoga() {
           </div>
           <div className="pl__na__right">
             <div className="pl__na__right__container">
-              <Title className="pl__na__coder__yoga__title">CODERYOGA</Title>
-              <Text className="pl__na__right__info__container">
+              <Title className="pl__na__coder__yoga__title" level={2}>
+                CODER-YOGA
```
Comment (paulstgermain) [inline src/components/pages/LandingPrograms/CoderYoga.js:30]: Is Coder-Yoga presented this way in our docs, or should it be 2 separate words? 🤔

Comment (paulstgermain) [reply src/components/pages/LandingPrograms/CoderYoga.js:30]: @dstrazzeri mentioned in a meeting just now that our docs refer to it as "CoderYoga" - maybe we should make this change before merging this code up!

File: src/styles/LandingPageStyles/index.less [hunk 4]
```diff
@@ -100,29 +87,54 @@
 }
 
 .landing-second-container-content {
-  text-align: center;
+  flex: 1;
   padding-top: 10%;
   margin-right: 5%;
+  text-align: center;
+  // background-color: #000;
```
Comment (paulstgermain) [inline src/styles/LandingPageStyles/index.less:94]: Let's remove this commented-out code if it isn't necessary!

**Review by paulstgermain — CHANGES_REQUESTED** [review]
This PR is looking great - the results speak for themselves, and for the most part the code you've written or touched on is formatted to match our current expectations!

That said, there are a couple things I'd love you to address before we merge this thing up - see the comments below for details!

**Review by paulstgermain — APPROVED** [review]

**Pull request merged**